crates/core/tests/dom.rs

```diff
@@ -61,6 +61,21 @@ fn dom_builder_example() {
     assert_eq!(buffer.as_str(), expected);
 }
 
+#[test]
+fn iframe_closing_tag_roundtrip() {
+    let orig_body = r#"<arb>
+    <iframe>
+    </iframe>
+</arb>"#;
+
+    let min_body = r#"<arb>
+    <iframe />
+</arb>"#;
```

Comment:
I wish there could be an extra newline at the beginning. The tags would be so much easier to read.